src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -200,9 +385,14 @@ private void register(VillagerCareer career)
             Validate.isTrue(career.profession == this, "Attempted to register career for the wrong profession.");
             career.id = careers.size();
             careers.add(career);
+            TRADES.put(career, new ArrayList<List<ITradeList>>());
```

Comment:
Having second lists associated with careers we control is just dumb. That's what the trade list is in the career object.